### PR TITLE
chore: improve http executor configs

### PR DESCRIPTION
**Pull request by bowenliang123**
# Checklist:

> [!IMPORTANT]  
> Please review the checklist below before submitting your pull request.

- [x] Please open an issue before creating a PR or link to an existing issue
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I ran `dev/reformat`(backend) and `cd web && npx lint-staged`(frontend) to appease the lint gods

# Description

- no behaviour changes
- rearrange the use of http executor configs properly

## Type of Change

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update, included: [Dify Document](https://github.com/langgenius/dify-docs)
- [x] Improvement, including but not limited to code refactoring, performance optimization, and UI/UX improvement
- [ ] Dependency upgrade

# Testing Instructions

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [ ] Test A
- [ ] Test B

### CODE REVIEW

File: api/core/workflow/nodes/http_request/http_request_node.py [hunk 1]
```diff
@@ -11,19 +12,16 @@
...
 from core.workflow.nodes.http_request.http_executor import HttpExecutor, HttpExecutorResponse
 from models.workflow import WorkflowNodeExecutionStatus
 
 HTTP_REQUEST_DEFAULT_TIMEOUT = HttpRequestNodeTimeout(
-    connect=min(10, MAX_CONNECT_TIMEOUT),
-    read=min(60, MAX_READ_TIMEOUT),
-    write=min(20, MAX_WRITE_TIMEOUT),
+    connect=min(10, dify_config.HTTP_REQUEST_MAX_CONNECT_TIMEOUT),
+    read=min(60, dify_config.HTTP_REQUEST_MAX_READ_TIMEOUT),
+    write=min(20, dify_config.HTTP_REQUEST_MAX_WRITE_TIMEOUT),
```
Comment (bowenliang123) [inline api/core/workflow/nodes/http_request/http_request_node.py:24]: I'm wondering why we use the minimum value for timeouts here? 
Minimum value will never make the  configs applicable. 
Shall we use the maximum value to allow it to be configurable and applicable?

cc previous related authors @laipz8200 @majian159

**Review by crazywoola — APPROVED** [review]
LGTM, please take a look as well @laipz8200

File: api/core/workflow/nodes/http_request/http_request_node.py (hunk 1 above)
Comment (laipz8200) [reply api/core/workflow/nodes/http_request/http_request_node.py:24]: I think using `max` here is ok.

Comment (bowenliang123) [reply api/core/workflow/nodes/http_request/http_request_node.py:24]: Shall we raise a separate PR to fix it? Let's merge this PR first.

Comment (laipz8200) [reply api/core/workflow/nodes/http_request/http_request_node.py:24]: I think it’s better to use `max` here, but this change shouldn’t be made in this PR.

I'll merge this PR first.

**Pull request merged**